Python/Product/Analysis/Interpreter/Ast/AstPythonMultipleMembers.cs

```diff
@@ -21,17 +21,42 @@
 using Microsoft.PythonTools.Analysis.Infrastructure;
 
 namespace Microsoft.PythonTools.Interpreter.Ast {
-    class AstPythonMultipleMembers : IPythonMultipleMembers, ILocatedMember {
+    class AstPythonMultipleMembers : IPythonMultipleMembers2, ILocatedMember {
         private readonly IReadOnlyList<IMember> _members;
-        private readonly Lazy<IList<IMember>> _resolvedMembers;
+        private IReadOnlyList<IMember> _resolvedMembers;
 
         public AstPythonMultipleMembers() {
             _members = Array.Empty<IMember>();
         }
 
         private AstPythonMultipleMembers(IReadOnlyList<IMember> members) {
             _members = members;
-            _resolvedMembers = new Lazy<IList<IMember>>(() => _members.Select(m => (m as ILazyMember)?.Get() ?? m).Where(m => m != this).ToArray());
+        }
+
+        private void EnsureMembers() {
+            if (_resolvedMembers != null) {
```

Comment:
Can it be called from multiple threads?

Reply:
It shouldn't be - this is only used in analysis.

Reply:
Actually, maybe through `Locations`, yeah.